package.json

```diff
@@ -4,16 +4,15 @@
   "main": "index.js",
   "private": true,
   "scripts": {
-    "build": "npm run remove-dist && npm run build:production && npm run build:static",
+    "build": "npm run build:production && npm run build:static",
     "build:production": "webpack -p --mode=production",
     "build:test": "webpack -p --mode=production --include-tests",
-    "build:dev": "npm run remove-dist && npm run build:static && webpack --mode=development --debug --devtool cheap-source-map --output-pathinfo",
+    "build:dev": "webpack --mode=development --debug --devtool cheap-source-map --output-pathinfo && npm run build:static",
     "build:static": "gulp svg imagemin",
     "dev": "npm run build:dev && npm run build:static",
     "dev-zip": "npm run build:dev && gulp release",
-    "watch": "npm run remove-dist && npm run build:static && webpack --watch --mode=development --debug --devtool cheap-module-eval-source-map --output-pathinfo",
+    "watch": "webpack --watch --mode=development --debug --devtool cheap-module-eval-source-map --output-pathinfo && npm run build:static",
```

Comment:
Not sure if we can do this with `--watch` as the `webpack` process won't exit until you stop the watch so it won't ever get to the `&&`. Can we move it to the beginning of the command or does that cause webpack to wipe the built assets?

Ideally, this added command wouldn't be necessary at all as static files should be built by webpack as well. Not sure if we have an issue for this but @tofumatt and I had spoke about this before. @tofumatt - if there isn't an issue for this, would you please create it? 🙏

Reply:
You're right—we talked about it but I never made the issue 😓

#1326 filed! 😄